Mathlib/Algebra/Group/Pointwise/Finset/Basic.lean

```diff
@@ -209,12 +209,13 @@ theorem card_inv_le : s⁻¹.card ≤ s.card :=
 theorem inv_empty : (∅ : Finset α)⁻¹ = ∅ :=
   image_empty _
 
-@[to_additive (attr := simp, aesop safe apply (rule_sets := [finsetNonempty]))]
+@[to_additive (attr := simp)]
 theorem inv_nonempty_iff : s⁻¹.Nonempty ↔ s.Nonempty := image_nonempty
 
 alias ⟨Nonempty.of_inv, Nonempty.inv⟩ := inv_nonempty_iff
```

Comment:
```suggestion
@[to_additive (attr := ...)]
alias ⟨Nonempty.of_inv, Nonempty.inv⟩ := inv_nonempty_iff
```
should work

Reply:
It looks like that tags both halves of it as `safe apply`, which will make aesop loop on these

Reply:
Oh right. I guess that works for cases where alias is used to generate one implication only